save-common/src/jvmMain/kotlin/org/cqfn/save/exceptions/BackendInternalException.kt

```diff
@@ -0,0 +1,10 @@
+package org.cqfn.save.exceptions
+
+import java.lang.Exception
+
+/**
+ * Exception for backend
+ */
+class BackendInternalException(message: String) : Exception(message) {
```

Comment:
Is it used anywhere?

Reply:
Not now, but I guess we will need it in the future